src/client.js

```diff
@@ -60,31 +60,10 @@ exports.Request = Request;
  */
 
 request.getXHR = () => {
-  if (
-    root.XMLHttpRequest &&
-    (!root.location ||
-      root.location.protocol !== 'file:' ||
-      !root.ActiveXObject)
-  ) {
```

Comment:
Don't we need `!root.location || root.location.protocol !== 'file:'` still?

Reply:
don't know... can bring it back...

Reply:
but really doe... why wouldn't it work on `file:`?
you might not be able to make request to another file located on the disk but you can still make cors request to other http request?

Reply:
I'd just keep it so this PR only drops the ActiveXObject stuff

Reply:
done